backend/app/configs/data_saver/csv_saver.yaml

```
@@ -4,4 +4,4 @@ defaults:
 
 name: csv_saver
 source: $kafka
-csv_file_path: ${oc.env:ROOT_PATH}/data_layer/database/db/csv_saver.csv
\ No newline at end of file
+csv_file_path: ${oc.env:ROOT_PATH}/app/data_layer/database/db/csv_saver.csv
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)